app/views/data_portal/index.html.erb

```diff
@@ -115,3 +119,31 @@
     </div>
   </div>
 </div>
+
+<script type="text/javascript">
+  function showFinancialDiaries() {
```

Comment:
This is a valid way to do this but we prefer to manage states with css classes. 
Logic would be way easier and so calculations